frontend/src/metabase/common/hooks/index.ts

```diff
@@ -1,6 +1,7 @@
 export * from "./entity-framework";
-export * from "./use-setting";
 export * from "./use-docs-url";
+export * from "./use-has-token-feature";
 export * from "./use-locale";
 export * from "./use-notification-channels";
-export * from "./use-has-token-feature";
+export * from "./use-setting";
+export * from "./use-temp-storage";
```

Comment:
I'd recommend adding `export * from "./use-url-with-utm";` from master here as well as it's a part of https://github.com/metabase/metabase/pull/48852/files#diff-59c462430aa91ffb6ce21259f0bda77a89a21c325d6b5c2c2826c2f98ea9469dR7 which wasn't backported, but that line is pretty safe since it just exports hooks from `use-url-with-utm` so that we could import that hook from `metabase/common/hooks` instead of having to use the exact file name. e.g.

https://github.com/metabase/metabase/blob/9b4d3d2b2ebb24975ed8a673f86e15d9417d1215/frontend/src/metabase/admin/upsells/components/use-upsell-link.ts#L1

Adding this line will also make this file matches master's version.